po/pl.po

```diff
@@ -21,363 +21,363 @@ msgstr ""
 #. create default categories
 #: src/database/Database.vala:941
 msgid "Groceries"
-msgstr ""
+msgstr "Pożywienie"
 
 #: src/database/Database.vala:942
 msgid "Fuel"
-msgstr ""
+msgstr "Paliwo"
 
 #: src/database/Database.vala:943
 msgid "Public transit"
-msgstr ""
+msgstr "Transport publiczny"
 
 #: src/database/Database.vala:944
 msgid "Restaurants"
-msgstr ""
+msgstr "Restauracje"
 
 #: src/database/Database.vala:945
 msgid "Entertainment"
-msgstr ""
+msgstr "Rozrywka"
 
 #: src/database/Database.vala:946 src/dialog/AddAccountDialog.vala:80
 msgid "Savings"
-msgstr ""
+msgstr "Oszczędności"
 
 #: src/database/Database.vala:947
 msgid "Personal care"
-msgstr ""
+msgstr "Higienia osobista"
 
 #: src/database/Database.vala:948
 msgid "Alcohol & Bars"
-msgstr ""
+msgstr "Alkohol i bary"
 
 #: src/database/Database.vala:949
 msgid "Emergency fund"
-msgstr ""
+msgstr "Na czarną godzinę"
 
 #. OK button
 #: src/view/TransactionView.vala:184 src/view/TransactionView.vala:889
 #: src/view/CategoryProperties.vala:144
 msgid "Apply"
-msgstr ""
+msgstr "Zatwierdź"
 
 #: src/view/TransactionView.vala:195 src/view/TransactionView.vala:341
 #: src/view/TransactionView.vala:905 src/view/TransactionView.vala:923
 msgid "Add transaction"
-msgstr ""
+msgstr "Dodaj transakcję"
 
 #: src/view/TransactionView.vala:318
 msgid "No results."
-msgstr ""
+msgstr "Brak wyników"
 
 #. Cancel button
 #: src/view/TransactionView.vala:345 src/view/CategoryProperties.vala:140
 #: src/dialog/AddAccountDialog.vala:121 src/dialog/AddCategoryDialog.vala:85
 #: src/widget/CellRendererCategoryPicker.vala:87
 msgid "Cancel"
-msgstr ""
+msgstr "Anuluj"
 
 #. columns
 #: src/view/TransactionView.vala:538
 msgid "Date"
-msgstr ""
+msgstr "Data"
 
 #: src/view/TransactionView.vala:551
 msgid "Merchant"
-msgstr ""
+msgstr "Klient"
```

Comment:
Using Google Translate i get Klient translated to customer.
In this case "Merchant" is the one selling the product.
I think using Klient might cause some confusion, but I don't speak Polish. 

Is this the correct term?

Reply:
"Merchant" in polish means "Kupiec", someone who mainly buys stuff and occasionally sells it.
I wanted to propose something more general, like "client", "subject", "entity", something that will fit most outcome and income possibilities. Would you call a merchant your employer from which you recieve your monthly payment?

Reply:
I think the best for polish version will be "Podmiot" = "Entity". This way it doesnt matter if you spent or earned money.

Reply:
You are right, it should be something like "subject", "second party" or, as HomeBank uses, "Payee".